### PR TITLE
Add prow test for Nordix/metal3-dev-tools

### PR DESCRIPTION
**Pull request by kashifest**
As a first step this PR enables shellcheck prow test.

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -232,6 +233,21 @@ periodics:
             secretName: github-token
 
 presubmits:
+  Nordix/metal3-dev-tools:
+  - name: shellcheck
```
Comment (tuminoid) [inline prow/config/config.yaml:237]: yaml lists need to be intended by two spaces. Please indent all lists (lines starting with `-`) by two from previous indent level.

**Review by tuminoid — CHANGES_REQUESTED** [review]

File: prow/config/config.yaml (hunk 1 above)
Comment (kashifest) [reply prow/config/config.yaml:237]: so whats our yamllint is checking then?

Comment (tuminoid) [reply prow/config/config.yaml:237]: It is only configured to check for trailing spaces, duplicate keys, and truthy values = it doesn't, the config sucks. But the next guy who wants to change the yaml who has yaml formatter configured in their editor will have huge diff.

Vscode has nice yaml plugin from RH you should absolutely install if you work with yaml files btw.

Comment (kashifest) [reply prow/config/config.yaml]: Thanks, done

Comment (kashifest) [reply prow/config/config.yaml]: I will push changes to yamllint workflow.

File: prow/config/plugins.yaml [hunk 2]
```diff
@@ -5,6 +5,29 @@ owners:
   skip_collaborators:
   - "metal3-io"
 plugins:
+  Nordix:
+    plugins:
+    - approve
```
Comment (tuminoid) [inline prow/config/plugins.yaml:10]: This file would also need yaml formatter.

Comment (kashifest) [reply prow/config/plugins.yaml]: Done

**Review by tuminoid — APPROVED** [review]
lgtm, but I'm no prow expert, leaving further review for others

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**